src/main/java/no/nav/pto/veilarbportefolje/auth/BrukerInnsynTilganger.java

```diff
@@ -0,0 +1,4 @@
+package no.nav.pto.veilarbportefolje.auth;
+
+public record BrukerInnsynTilganger(boolean tilgangTilKode6, boolean tilgangTilKode7, boolean tilgangTilEgenAnsatt) {
```

Comment:
Since you are refactoring this part maybe we can change:
kode 6 -> Strengt fortrolig
kode 7 -> Fortrolig
egenAnsatt -> Skjermet
maybe add kode 19 -> Strengt fortolig utland